joern-cli/frontends/jimple2cpg/src/main/scala/io/joern/jimple2cpg/Main.scala

```diff
@@ -23,7 +37,22 @@ private object Frontend {
       programName("jimple2cpg"),
       opt[String]("android")
         .text("Optional path to android.jar while processing apk file.")
-        .action((android, config) => config.withAndroid(android))
+        .action((android, config) => config.withAndroid(android)),
+      opt[Unit]("full-resolver")
+        .text("enables full transitive resolution of all references found in all classes that are resolved")
+        .action((_, c) => c.withFullResolver(true)),
```

Comment:
Also, try to use consistent naming with these parameters. This one is just `c` instead of `config.